worker/peergrouper/worker_test.go

```diff
@@ -731,7 +750,11 @@ func mustNext(c *gc.C, w *voyeur.Watcher) (val interface{}) {
 		c.Logf("mustNext %p", w)
 		ok := w.Next()
 		val = w.Value()
-		c.Logf("mustNext done %p, ok: %v, val: %#v", w, ok, val)
+		if ok {
+			members := val.([]replicaset.Member)
+			val = "\n" + prettyReplicaSetMembers(members)
```

Comment:
This looks problematic! `val` is being updated to improve the log output but the modified val is also being returned via `voyerResult`. I'm pretty sure that's not intended.